pkgs/tools/networking/hss/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
+  name = "hss-${version}";
+  version = "1.0.1";
+
+  inherit ruby;
+  gemfile  = ./Gemfile;
+  lockfile = ./Gemfile.lock;
+  gemset   = ./gemset.nix;
```

Comment:
Fair enough, this seems simpler and works just the same. Thanks for the feedback :)